src/py42/exceptions.py

```diff
@@ -275,25 +262,23 @@ class Py42BadRestoreRequestError(Py42BadRequestError):
     a bad request."""
 
     def __init__(self, exception):
-        message = u"Unable to create restore session."
-        super(Py42BadRestoreRequestError, self).__init__(exception, message)
+        message = "Unable to create restore session."
+        super().__init__(exception, message)
 
 
 class Py42InvalidPageTokenError(Py42BadRequestError):
     """An error raised when the page token given is invalid."""
 
     def __init__(self, exception, page_token):
-        message = u"Invalid page token: {}".format(page_token)
-        super(Py42InvalidPageTokenError, self).__init__(exception, message)
+        message = f"Invalid page token: {page_token}"
```

Comment:
This message would be improved with quotes around `{page_token}` and a period at the end